spec/ddtrace/contrib/dalli/instrumentation_spec.rb

```diff
@@ -18,7 +18,9 @@ def all_spans
 
   # Enable the test tracer
   before(:each) do
-    Datadog.configure { |c| c.use :dalli }
```

Comment:
Tiny nitpick - feel free to ignore. Looks like this change is unneccessary.

Reply:
It is technically unnecessary. Do you prefer I remove it?

Reply:
I feel like these one liners look better in concise form.
But this is only slight preference, so it's up to you :)

Reply:
I agree, it is more concise in that one line form.